File: 프로그래머스/lv0/120830. 양꼬치/양꼬치.py
def solution(n, k):
    food_price = 12000 * n
    
    if n >= 10:
        juice_price = 2000 * (k - n // 10)
    else:
        juice_price = 2000 * k
        
    answer = food_price + juice_price
    return answer

File: 프로그래머스/lv0/120830. 양꼬치/양꼬치.js
function solution(n, k) {
    var answer = 0;
    
    if(n < 10) {
        answer = 12000 * n + 2000 * k;
        return answer;
    } else if (n >= 10) {
        answer = 12000 * n + (k - Math.floor(n / 10)) * 2000;
        return answer;
    }
}